test/integration/db_integration_test.go

```diff
@@ -77,7 +91,7 @@ func TestIntegrationWithDb_GetRecords(t *testing.T) {
 
 	// last record with the biggest pulse number won't be processed, so we do not expect this record in DB
 	expRecordsCount := recordsInPulse * (pulsesNumber - 1)
-	ts.WaitRecordsCount(t, expRecordsCount, 6000)
+	ts.WaitRecordsCount(t, expRecordsCount+1, 6000)
```

Comment:
```
expRecordsCount := recordsInPulse * pulsesNumber
ts.WaitRecordsCount(t, expRecordsCount, 6000)
```
is it possible to make this change?

Reply:
Yes, but i do not know what do you wait in this test, but it is pulses*perpulse.